app/models/katello/content_view_history.rb

```diff
@@ -66,5 +66,13 @@ def humanized_status
         _("Success")
       end
     end
+
+    def description
+      if self.task.try(:label) == "Actions::Katello::ContentView::Publish"
+        content_view_version.description
```

Comment:
I think the migration of content_view_version.description -> content_view_history.notes could be tricky and I think the amount of work would depend on the implementation. Thinking it through, we could either:

1. Assume that all content view versions have a content view history for their publish. We could raise an exception if they don't and let the user deal with it.
2. Assume that maybe some versions don't have a cv history for publish and create it. This is a bit tricky because content view histories [rely on the task label](https://github.com/Katello/katello/blob/master/app/models/katello/content_view_history.rb#L45) so we'd either need to create that task by hand or create a new field on cv history to identify its type (publish, promotion, removal, etc).

I think 1 is fairly trivial but the second option may not be. Maybe there's also another option I am overlooking? Thoughts?